unittests/failuretests.py

```diff
@@ -150,10 +150,9 @@ def test_extraframework_dependency_method(self):
 
     def test_sdl2_notfound_dependency(self):
         # Want to test failure, so skip if available
-        if shutil.which('sdl2-config'):
-            raise unittest.SkipTest('sdl2-config found')
-        self.assertMesonRaises("dependency('sdl2', method : 'sdlconfig')", self.dnf)
-        if shutil.which('pkg-config'):
+        if not shutil.which('sdl2-config'):
+            self.assertMesonRaises("dependency('sdl2', method : 'sdlconfig')", self.dnf)
+        if not shutil.which('pkg-config'):
```

Comment:
So... Apparently this was always broken? This deserves a call-out in the commit message.

Reply:
Does the revised commit message look better?

Reply:
Ah, I was thinking about the fact that this test only passes if sdl2-config skips... because if it gets to the pkg-config check, it only runs when pkg-config *is* found, and then it asserts that the dependency must not be found, which seems... confusing.